router/xgress_geneve/listener.go

```diff
@@ -0,0 +1,108 @@
+/*
+	Copyright 2019 NetFoundry, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	https://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package xgress_geneve
+
+import (
+	"encoding/binary"
+	"net"
+	"syscall"
+
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+	"github.com/michaelquigley/pfxlog"
+	"github.com/openziti/fabric/router/xgress"
+)
+
+type listener struct{}
+
+func (self *listener) Listen(string, xgress.BindHandler) error {
+	go func() {
+		log := pfxlog.Logger()
+		// Open UDP socket to listen for Geneve Packets
+		conn, err := net.ListenPacket("udp", ":6081")
+		if err != nil {
+			log.WithError(err).Errorf("failed to open geneve interface - udp")
+			// error but return gracefully
+			return
+		} else {
+			log.Infof("geneve interface started successfully - udp: %s", conn.LocalAddr().String())
+		}
+		// Close it when done
+		defer conn.Close()
+		// Open a raw socket to send Modified Packets to Networking Stack
+		fd, err := syscall.Socket(syscall.AF_INET, syscall.SOCK_RAW, syscall.IPPROTO_RAW)
+		if err != nil {
+			log.WithError(err).Errorf("failed to open geneve interface - fd")
+			// error but return gracefully
+			return
```

Comment:
Can remove the `else` from around the log statement

Reply:
good point, will eliminate the else